platform/mellanox/mlnx-platform-api/sonic_platform/thermal.py

```diff
@@ -420,3 +437,123 @@ def get_high_critical_threshold(self):
         if self.category == THERMAL_DEV_CATEGORY_MODULE and value_float == THERMAL_API_INVALID_HIGH_THRESHOLD:
             return None
         return value_float / 1000.0
+
+
+    @classmethod
+    def _write_generic_file(cls, filename, content):
```

Comment:
what the purpose of the generic file? can you please add more description for it?

Reply:
Updated description.